ddtrace/contrib/internal/flask/patch.py

```diff
@@ -224,11 +224,18 @@ def patch():
     _w("flask.templating", "_render", patched_render)
     _w("flask", "render_template", _build_render_template_wrapper("render_template"))
     _w("flask", "render_template_string", _build_render_template_wrapper("render_template_string"))
+    # werkzeug >= 2.1.0
     try:
         _w("werkzeug.debug.tbtools", "DebugTraceback.render_debugger_html", patched_render_debugger_html)
     except AttributeError:
         log.debug("Failed to patch DebugTraceback.render_debugger_html, not supported by this werkzeug version")
 
+    # werkzeug <= 2.0.3
```

Comment:
can we check here the flag _iast_enabled and only do the patch if it's true?

Reply:
This is a `core.dispatch`, a common handler. The function that listens to it is in `iast_listeners` and is loaded when IAST is enabled:

https://github.com/DataDog/dd-trace-py/blob/main/ddtrace/appsec/_iast/processor.py#L20